docs/get-started/v2/get-started/features/noise-cancellation.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Noise Cancellation
+nav: 3.9
+---
+
+The Noise cancellation feature is an invaluable tool designed to enhance the audio quality in scenarios such as conferences, live streams, and recordings where unwanted background noise can degrade the listening experience. By leveraging advanced Artificial Intelligence (AI) algorithms, this feature intelligently identifies and suppresses extraneous sounds, ensuring clear and crisp audio output.
+
+## Key Benefits
```

Comment:
This is fluff piece which is usually not necessary in technical documentations. Point out any key benefits or features, if any.

Reply:
Yes, Felt so will remove second line and just keep first one. 
Key benefits are added below this line